Automattic/jetpack
# [Share this post panel] Add social media follower counts

#### Changes proposed in this Pull Request:
This PR adds social media follower counts next to the checkboxes in the Share this post panel. It also shows a Fix link when there currently are no follower counts being reported. 

This is the case for LinkedIn, which needs a reauthentication because it needs an additional API scope. Disconnecting & reconnecting the account is the fastest fix for this.

| Count present | Count not present |
|---------------|--------------------|
| ![CleanShot 2022-12-09 at 14 06 33@2x](https://user-images.githubusercontent.com/528287/206709268-734b73f1-3a1c-46e1-86e7-0c604a7ffbde.png) | ![CleanShot 2022-12-09 at 14 09 33@2x](https://user-images.githubusercontent.com/528287/206709681-f77d941f-30a5-435b-b88c-e06d6641e1ac.png) |

When clicking the link, the Jetpack social connection management page on WordPress.com is opened. Here the user can disconnect & reconnect LinkedIn. When closing this window, the connections are refreshed and the count will show.

#### Jetpack product discussion
pdDOJh-ZI-p2
pdDOJh-Yp-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1. Apply this PR
2. Create a post and verify that the counts are shown in the pre-publish panel.

Retrieving the LinkedIn counts:
- Apply this patch to your sandbox: D94044-code
- Sandbox `public-api.wordpress.com`
- Click the fix link, disconnect & connect LinkedIn
- You should be presented by an authentication screen that asks for "Use your 1-st degree connection data"
- When closing the window, the connections will refresh & the count will show (this can take some time on a sandboxed connection)

To reset back to "Fix":
- Unsandbox `public-api.wordpress.com`
- Click the `Connect an account` link, disconnect & connect LinkedIn
